samples/matter/smoke_co_alarm/sample.yaml

```diff
@@ -0,0 +1,34 @@
+sample:
+  description: Matter Smoke CO Alarm example
+  name: Matter Smoke CO Alarm
+tests:
+  sample.matter.smoke_co_alarm.debug:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf54l15pdk/nrf54l15/cpuapp
+    platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp nrf54l15pdk/nrf54l15/cpuapp
+    tags: sysbuild ci_samples_matter
+  sample.matter.smoke_co_alarm.release:
+    sysbuild: true
+    build_only: true
+    extra_args: FILE_SUFFIX=release
+      SB_CONFIG_BOOTLOADER_MCUBOOT=y SB_CONFIG_PARTITION_MANAGER=y
```

Comment:
do we need these configs for release configuration?

Reply:
No, this seems to be leftover, but it's also present in all other samples, so let's remove in consistently in another PR.